src/vs/editor/contrib/hover/browser/hover.css

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-editor .hoverHighlight {
```

Comment:
Since this is a new file, you'll need to import it like this:

https://github.com/microsoft/vscode/blob/1d70080dee6ad46ca5bb1b7583eaa082e2d235be/src/vs/base/browser/ui/hover/hoverWidget.ts#L11